app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,436 @@
+---
+title: Self-service registration for higher education institutions and their school direct partners
+description: Exploring ways to help higher education institutions register and invite their school direct partners to use the Manage service
+date: 2021-03-18
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+How might we provide a self-service journey for higher education institutions (HEIs) and their school direct partners (SDs) to register on the Manage service?
```

Comment:
Change to a normal intro - not a question.

Reply:
I have updated as per your suggestion